gulp/util.js

```diff
@@ -224,14 +228,18 @@ function filterNonCodeFiles() {
 
 // builds the theming related css and provides it as a JS const for angular
 function themeBuildStream() {
-  return gulp.src(config.themeBaseFiles.concat(path.join(config.paths, '*-theme.scss')))
-      .pipe(concat('default-theme.scss'))
-      .pipe(utils.hoistScssVariables())
-      .pipe(sass())
-      .pipe(dedupeCss())
-      // The PostCSS orderedValues plugin modifies the theme color expressions.
-      .pipe(minifyCss({ orderedValues: false }))
-      .pipe(utils.cssToNgConstant('material.core', '$MD_THEME_CSS'));
+  var paths = config.themeBaseFiles;
+  config.componentPaths.forEach(component => paths.push(path.join(component, '*-theme.scss')));
+  paths.push(config.themeCore);
```

Comment:
This fixes the issue where some apps had a white background instead of light gray in the presubmits.